contracts/governance/extensions/GovernorCountingOverridable.sol

```diff
@@ -9,7 +9,10 @@ import {GovernorVotes} from "./GovernorVotes.sol";
 
 /**
  * @dev Extension of {Governor} which enables delegatees to override the vote of their delegates. This module requires a
- * token that inherits {VotesExtended}.
+ * token that inherits `VotesExtended`.
+ *
+ * NOTE: Overriding a vote doesn't force the vote to be casted. Consider that an overridden vote won't be counted towards
```

Comment:
That is incorrect. The overriden vote do affect the result, and count toward quroum even if the delegate never casts a vote.